addons/mail/models/mail_render_mixin.py

```diff
@@ -226,11 +230,11 @@ def _prepend_preview(self, html, preview):
             preview = preview.strip()
 
         if preview:
-            html_preview = f"""
+            html_preview = Markup("""
                 <div style="display:none;font-size:1px;height:0px;width:0px;opacity:0;">
-                  {tools.html_escape(preview)}
+                   {}
```

Comment:
Mmmh ? Strange formatting isn't it ? Can't we use standard name-based format ?

Reply:
Do you mean `%`? Switching to str.format was just less change from the original f-string.